draft-looker-jwt-cwt-status-list.md

```diff
@@ -32,38 +32,57 @@ This specification defines a status list representation and processing rules for
+This document defines a Status List in JWT and CWT representation that describes the individual statuses of multiple Subject Tokens. The Status List contains binary data, where each Subject Token corresponds with a certain position within a bit array(also known as bit vector, string or list). The document also defines how an issuer of a Subject Token in JWT or CTW representation references a Status List to convey status data like suspension or revocation of the Subject Token and related information. Status Lists may be composed for a range of various Status Types with one or more bits, the document defines two configurations for the most common use cases as well as an extensibility mechanism for custom Status List Types. The Status List may be used by a Status List Provider in the Issuer-Holder-Verifier model, as described in {{RFC-XXXX}} to express the status of verifiable credentials (Subject Tokens) issued by an issuer.
 
 The following diagram depicts the basic conceptual relationship.
 
 ~~~ ascii-art
 
 +---------------+                     +---------------+
-|               |                     |               |
-|               |                     |               |
-|     Token     |    References       |  Status List  |
-|(JWT/CWT Based)|-------------------->|(JWT/CWT Based)|
-|               |                     |               |
-|               |                     |               |
+|               |      References     |               |
+|               |-------------------->|               |
+| Subject Token |                     |  Status List  |
+|(JWT/CWT Based)|                     |(JWT/CWT Based)|
+|               |  Describes Status   |               |
+|               |<--------------------|               |
 +---------------+                     +---------------+
 
 ~~~
 
+## Rationale
+
+Revocation mechanisms are an essential part for most identity ecoosystems. In the past, revocation of X.509 TLS certificates has been proven difficult as traditional certificate revocation lists (CRLs) have limited scalability and the Online Certificate Status Protocol (OCSP) has additional privacy risks as the client is leaking the requested website to a third party. OSCP stapling is adressing some of these problems at the cost of less up-to-date data. Modern approaches use accumulator-based revocation registries and Zero-Knowledge-Proofs to accomodate for this privacy gap but face scalability issues again.
+
+The approach of this specification seeks to find a balance between scalability, security and privacy by minimizing the status information to mere bits and compressing the resulting binary data. Thereby a Status List may contain statuses of 100.000 or more Subject Tokens, but still remain relatively small. Placing large amounts of Subject Tokens into the same list also enables a herd privacy towards the Issuer or Status List Provider.
```

Comment:
Is the point that 100000 or more subject tokens makes it reasonable privacy preserving?